settings/R/check.all.settings.R

```diff
@@ -20,10 +20,10 @@ check.inputs <- function(settings) {
   }
   
   # get list of inputs associated with model type
-  dbcon <- db.open(settings$database$bety)
-  on.exit(db.close(dbcon))
+  dbcon <- PEcAn.DB::db.open(settings$database$bety)
```

Comment:
For now this works (especially since I haven't fixed `betyConnect` like I said I would) but we really should be switching over to `dplyr` functions. Essentially any time we use `db.query` with `SELECT` we can replace that with the `dplyr` functions `tbl` and `filter` and `select`.  I can tackle the full transition after AGU/Christmas but the sooner people get used to the `dplyr` functions, the better.